jobs/config.json

```diff
@@ -9782,7 +9698,6 @@
       "--env-file=jobs/platform/gce.env",
       "--env-file=jobs/periodic-kubernetes-e2e-kubeadm-gce-1-7.env",
       "--extract=ci/latest-1.7",
-      "--gcp-project=k8s-jkns-gci-gke-serial-1-4",
```

Comment:
@luxas see here, previously I tossed a random gke project for kubeadm-1.7 job, so I would assume the job should work on default quota. I'll fix the flag issue and see if it will be fine.